jdat_notebooks/MRS_Mstar_analysis/JWST_Mstar_dataAnalysis_analysis.ipynb

```diff
@@ -20,7 +20,7 @@
     "**Data:** Simulated [MIRI MRS](https://jwst-docs.stsci.edu/mid-infrared-instrument/miri-observing-modes/miri-medium-resolution-spectroscopy) spectrum of AGB star.<br>\n",
```

Comment:
<p><code>jdaviz</code> configurations should be imported at the beginning of the notebook, not just before you open them.</p><p> </p><p>If you're concerned about having multiple <code>specviz</code> instances open, you should just be able to use different variable names (e.g., <code>specviz = Specviz()</code> the first time, <code>specviz2 = Specviz()</code> the second time).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/dat_pyinthesky/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>